lucene/misc/src/java/org/apache/lucene/misc/index/MultiPassIndexSplitter.java

```diff
@@ -101,7 +111,7 @@ public void split(IndexReader in, Directory[] outputs, boolean seq) throws IOExc
           .setOpenMode(OpenMode.CREATE));
       System.err.println("Writing part " + (i + 1) + " ...");
       // pass the subreaders directly, as our wrapper's numDocs/hasDeletetions are not up-to-date
-      final List<? extends FakeDeleteLeafIndexReader> sr = input.getSequentialSubReaders();
+      final List<? extends FakeDeleteLeafIndexReader> sr = input.getSequentialSubReadersWrapper();
```

Comment:
BaseCompositeReader.getSequentialSubReaders() is a protected method so it cannot be directly called from here. A wrapper method is needed to move this class to `o.a.l.misc.index` from `o.a.l.index`.